lib/localization/l10n/intl_messages_en.arb

```diff
@@ -24,5 +24,10 @@
   "reserveSlotButtonTitle": "Reserve Slot",
   "createReservationSuccessSnackbar": "Slot reserved",
   "createReservationFailedDialogTitle": "Warning",
-  "createReservationFailedDialogDescription": "Your reservation could not be created."
+  "createReservationFailedDialogDescription": "Your reservation could not be created.",
+  "aboutTheAppTitle": "About the App",
```

Comment:
Should we just call this aboutTitle and "About"? (web support)

Reply:
I personally think about the app sounds better than just about, especially when translated to German